packages/@o3r/eslint-plugin/tsconfig.spec.json

```diff
@@ -1,5 +1,9 @@
 {
   "extends": "../../../tsconfig.jest",
+  "compilerOptions": {
+    "moduleResolution": "NodeNext",
```

Comment:
Why did you need it, it is generally not compatible with TS reviver (as it inforce to add the `.js` in the imports in the code :S